### ISSUE TITLE
Flexrex install broken for v0.10.0 and v0.10.1

### ISSUE THREAD

**Issue by miry** [codedellemc/rexray#1024]
I could not install flexrex after upgrade to 0.10.* on AWS and Centos 7

# Bug Reports

```
# rexray flexrex install
INFO[0000] updated log level                             logLevel=debug
DEBU[0000] created scoped scope                          new=rexray.cli parentScopes=rexray,
DEBU[0000] os.args                                       time=1505224602739 val=[rexray flexrex install]

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x17f1909]

goroutine 1 [running]:
github.com/codedellemc/rexray/cli.(*CLI).installEmbeddedScript(0xc420406000, 0x3276000, 0xc42039a120, 0x0, 0x0, 0xc420011320, 0xc420062901, 0xc420487938)
	/go/src/github.com/codedellemc/rexray/cli/cmds_50_scripts.go:173 +0x29
github.com/codedellemc/rexray/cli.(*CLI).installScripts(0xc420406000, 0x3276000, 0xc42039a120, 0xc4204879e8, 0x1, 0x1, 0xa, 0x16c24d0, 0xc42040cb40, 0x3301880, ...)
	/go/src/github.com/codedellemc/rexray/cli/cmds_50_scripts.go:129 +0xb2
github.com/codedellemc/rexray/cli.(*CLI).initFlexRexCmds.func2(0xc42040f680, 0x3301880, 0x0, 0x0)
	/go/src/github.com/codedellemc/rexray/cli/cmds_51_flexrex.go:41 +0x325
github.com/codedellemc/rexray/vendor/github.com/spf13/cobra.(*Command).execute(0xc42040f680, 0x3301880, 0x0, 0x0, 0xc42040f680, 0x3301880)
	/go/src/github.com/codedellemc/rexray/vendor/github.com/spf13/cobra/command.go:654 +0x299
github.com/codedellemc/rexray/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4203b2d80, 0xc4201fdc48, 0xc4201fdc40, 0xc420054040)
	/go/src/github.com/codedellemc/rexray/vendor/github.com/spf13/cobra/command.go:729 +0x339
github.com/codedellemc/rexray/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4203b2d80, 0x2631710, 0xc420406000)
	/go/src/github.com/codedellemc/rexray/vendor/github.com/spf13/cobra/command.go:688 +0x2b
github.com/codedellemc/rexray/cli.(*CLI).Execute(0xc420406000)
	/go/src/github.com/codedellemc/rexray/cli/cli.go:246 +0x55
github.com/codedellemc/rexray/cli.Execute(0x3276000, 0xc42039a120, 0x0, 0x0)
	/go/src/github.com/codedellemc/rexray/cli/cli.go:229 +0x57
main.main()
	/go/src/github.com/codedellemc/rexray/rexray.go:188 +0x7d6
```

## Version

```shell
# rexray version
REX-Ray
-------
Binary: /usr/bin/rexray
Flavor: client+agent+controller
SemVer: 0.10.1
OsArch: Linux-x86_64
Commit: 6e8043ac569707b0fb3e776309769742ca09e174
Formed: Mon, 11 Sep 2017 15:05:01 UTC
```

## Expected Behavior
Kubernetes integration https://rexray.readthedocs.io/en/stable/user-guide/schedulers/#kubernetes

## Actual Behavior
Crashed

## Steps To Reproduce
Please list the steps to reproduce the issue in this section.

1. The first step should always be enabling `debug` logging.
  * Open the file `/etc/rexray/config.yml`
  * Set the log level for both REX-Ray and libStorage to `debug` and enable
HTTP request and response tracing for libStorage:
```yaml
rexray:
  logLevel:        debug
libstorage:
  logging:
    level:         debug
    httpRequests:  true
    httpResponses: true
```
* Please list each step with as much detail as possible.
* The more information gathered up front, the easier it is to solve
the problem.
* Thank you!

## Configuration Files
Please paste any related configuration files, such as `/etc/rexray/config.yml`
in this section. Please use the appropriate formatting when pasting YAML.
content. For example:

```yaml
libstorage:
  service: ebs
  integration:
    volume:
      operations:
        create:
          default:
            size: 50
ebs:
  accessKey:
  secretKey:
  region:
```

**Comment by akutz** [comment]
Hi @miry,

I'm sorry about this. I'm looking into it.

**Comment by akutz** [comment]
Hi @miry,

There's a fix building at #1025. Once it's merged if you wish to build and test yourself you can just `go get -u github.com/codedellemc/rexray` and it will update your local REX-Ray binary to the new version.

However, if you wish to build the fix from this PR branch you could use Docker to so in a single line. Every Travis-CI build for REX-Ray [uploads](https://travis-ci.org/codedellemc/rexray/jobs/274639158#L963-L975) a Gist with [instructions](https://gist.github.com/anonymous/e24d8ac00f688aca907d50ef2517f88c) on how to use Docker to reproduce a build in a single line:

```bash
docker run -it \
  -v "$(pwd)":"/tmp/rexray" \
  golang:1.8.3 \
  bash -c "git clone https://github.com/codedellemc/rexray \
      \"/go/src/github.com/codedellemc/rexray\" &&
    cd \"/go/src/github.com/codedellemc/rexray\" && 
    git fetch origin +refs/pull/1025/merge: &&
    git fetch --tags origin &&
    git checkout -b aa4ec73 aa4ec736c03b9f8764b8918cf754e8a363b49ad0 &&
    XGOOS=linux XGOARCH=amd64 GOOS= GOARCH= go generate &&
    GOOS=linux GOARCH=amd64 go build -o \"rexray\" &&
    cp -f \"rexray\" \"/tmp/rexray\"" && \
  md5sum "rexray" && \
  ls -al "rexray"
```

The above command will emit:

```bash
Cloning into '/go/src/github.com/codedellemc/rexray'...
remote: Counting objects: 28729, done.
remote: Compressing objects: 100% (58/58), done.
remote: Total 28729 (delta 23), reused 53 (delta 17), pack-reused 28649
Receiving objects: 100% (28729/28729), 45.25 MiB | 9.67 MiB/s, done.
Resolving deltas: 100% (12351/12351), done.
Checking connectivity... done.
remote: Counting objects: 8, done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 8 (delta 6), reused 7 (delta 6), pack-reused 0
Unpacking objects: 100% (8/8), done.
From https://github.com/codedellemc/rexray
 * branch            refs/pull/1025/merge -> FETCH_HEAD
Switched to a new branch 'aa4ec73'
ee96663719be07d14ea1405a4c67a906  rexray
-rwxr-xr-x  1 akutz  wheel    96M Sep 12 10:01 rexray*
```
